### PR TITLE
MH-12980, Update documentation landign page

### PR DESCRIPTION
**Pull request by lkiesow**
- Fix a few links
- Explain mailing list subscriptions
- Remove pullrequest filter

### CODE REVIEW

**Comment by gregorydlogan** [comment]
Looks reasonable to me.  Do you want to backport this to 5.x and then bring it forward from there?

**Comment by lkiesow** [comment]
No, it's deployed from `develop` anyway and no one should be using this standalone.

**Comment by staubesv** [comment]
Hi @lkiesow 

Looking at ...

screen shot 2018-07-06 at 12 50 13

... I would recommend to be consistent in the list of mailing lists in means of always provide the list description first, e.g. "Security announcements (low frequency)" instead of "Low frequency. Security announcement".

There is also a small inconsistency considering the description below the list: The list opencast-es@opencastproject.es is not a list you "You can subscribe via Google Groups (linked above)" - but that is not related to this PR, just mentioning it.

For me I'm fine to accept this PR as is although I would recommend to do formulate "low frequency" as described above.

Shall I merge or do you want to follow the recommendation?

**Comment by staubesv** [comment]
... in case you would touch this again anyway: "Users list." is not a sentence and therefore does not require a full stop. All the trailing fullstops in the list items could be removed. Looks better ;-)

File: docs/guides/index.html [hunk 1]
```diff
@@ -103,68 +102,71 @@ <h2><span class="glyphicon glyphicon-wrench" aria-hidden="true"></span> Develope
...
+            <li><a href="https://groups.google.com/a/opencast.org/forum/#!forum/users">
+                users@opencast.org</a> –
+                Users list.</li>
+            <li><a href="https://groups.google.com/a/opencast.org/forum/#!forum/dev">
+                dev@opencast.org</a> –
+                Developers list.</li>
+            <li><a href="https://groups.google.com/a/opencast.org/forum/#!forum/anwender">
+                anwender@opencast.org</a> –
+                German speaking community list.</li>
+            <li><a href="mailto:opencast-es@opencastproject.es">
```
Comment (staubesv) [inline docs/guides/index.html]: E-mail or Google forum?

**Comment by lkiesow** [comment]
I've fixed the format.

I'm going to talk to Carlos about the correct format for the Spanish community list and will fix that separately once I figured out where it lives and how to subscribe.

**Comment by staubesv** [comment]
Thanks! Probably, it is a mailing list, see https://opencast.org/communication (also just an E-mail address). I'm fine with not addressing this here and now.

**Review by staubesv — APPROVED** [review]

**Pull request merged**